extensions/amp-story/0.1/sources.js

```diff
@@ -48,6 +50,31 @@ export class Sources {
       element.setAttribute('src', this.srcAttr_);
     }
 
+    // Wait for "loadedmetadata" before adding tracks
+    // Firefox issue workaround
```

Comment:
Can you add more insight on what the issue is in Firefox?

Reply:
yep sure